website_cookiebot/wizards/res_config_settings.py

```diff
@@ -15,6 +15,7 @@ class ResConfigSettings(models.TransientModel):
     )
     cookiebot_enabled = fields.Boolean(
         string="Cookiebot",
+        related="website_id.cookiebot_enabled",
         compute="_compute_cookiebot_enabled",
```

Comment:
Remove this attribute

Reply:
It's used as an onchange to set it off when the Odoo's own cookies setting is on

Reply:
Then just remove the related and put an inverse for clearer understanding.

Reply:
Sorry, I missread it. It was useless...